src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -76,9 +76,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
```

Comment:
Might want to include a more detailed function description above. Like it seems like there's some text that is getting replaced by some image? But it seems a little unclear why and what is going on because in the doPost method description it only says "stores a new {@link Message}."